Homework lesson19 added.

iBubbleGun opened the pull request:
Added homework lesson19.
Please, check it.

iBubbleGun committed:
Added homework lesson 19.

iBubbleGun committed:
Updated .gitignore

iBubbleGun committed:
Added homework lesson19.

iBubbleGun committed:
Merge branch 'lesson19' of https://github.com/iBubbleGun/Hillel into lesson19

kartius commented on src/main/java/edu/hillel/homework/lesson19/Main.java:
```diff
@@ -0,0 +1,64 @@
+                        bw.write("Max-size=2048");
+                        bw.newLine();
+                        bw.write("Format=[TIMESTAMP] [LEVEL] Message: [MESSAGE]");
+                        bw.newLine();
```
Please, move this internal logic to separated method

iBubbleGun replied:
Updated.

iBubbleGun committed:
Updated homework lesson19.
The method for generating the default configuration file has been moved to a separate class.
